config.py

```diff
@@ -375,6 +375,23 @@ def __init__(self):
 
         # NOTE(norswap): The pprof server listens on port 6060 by default.
 
+        # ==========================================================================================
+        # Account Abstraction Configuration
+
+        # === Private Key ===
+
+        self.deployer_key = None
+        """
+        Private key to use for deploying 4337 contracts and paymaster signature (None by default).
+        Will be used if set, otherwise will prompt users to enter private key.
+        """
```

Comment:
Is it good to always make the deployer the same as the paymaster signature address? Haven't really thought about this in earnest.

I know people like to talk about "secure signers" as option for holding live keys (e.g. paymaster signer) and it might be a bit of a pain to use such a signer for deployment? Is that a good reason to separate them?

Reply:
This is a good question, let's revisit it: https://github.com/0xFableOrg/simple-op-stack-rollup/issues/40